src/test/resources/runners/suiteAddAndSubtractionTest.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+<suite name="SuiteAddAndSubtractTests">
+    <test thread-count="5" name="Test">
+        <groups>
+            <run>
+                <include name="subAddition"></include>
+            </run>
+        </groups>
+        <classes>
+            <class name="com.epam.tc.hw1.AdditionCalculatorTest"></class>
+            <class name="com.epam.tc.hw1.SubtractionCalculatorTest"></class>
+            <class name="com.epam.tc.hw1.TestingCalculatorOperations"></class>
+        </classes>
+    </test>
+</suite>
```

Comment:
Please add a new blank line at the end of the file
![image](https://user-images.githubusercontent.com/42721087/210800055-4d17969c-27ab-4326-b266-15addb252abe.png)